conf/convert_document.yaml

```diff
@@ -0,0 +1,52 @@
+defaults:
+  - _self_
+  - llm: gpt4o
+
+environment:
+  _target_: tapeagents.environment.ToolCollectionEnvironment
+  tools:
+    - _target_: tapeagents.tools.document_reader.DocumentReader
+      preferred_pdf_converter:
+        _target_: hydra.utils.get_class
```

Comment:
what does this util do? I would expect just _target_: tapeagents.tools.converters.PdfConverter